### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by Solobarine**
This is a pull request to review the styles added to the bookstore. Enjoy and drop your thoughts.

### CODE REVIEW

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Solobarine, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [ ] React inline styling is allowed but a variable should be created and passed to React style, kindly consider implementing this instead of using ` <button type="button" style={{ display: 'none' }} onClick={createApp}>create app</button>` you can create a variable with the styles and assign it to the style attribute
e.g
```javascript
const style = {
   display: none,
}

<button type="button" style={style} onClick={createApp}>create app</button>
..
```
- [ ] Your app should be identical to the Zepline design, kindly consider implementing this so your app follows the design guidelines

|Your app|Zeplin design|
|-|-|
|Screenshot_20221112_042341|Screenshot_20221110_210233|
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 1]
```diff
@@ -10,10 +10,49 @@ li {
...
+.book button {
+  padding: 7px;
+  color: #999;
+  background-color: #5f5;
+  border-radius: 10px;
+}
+
+.sectionTwo {
+  padding-top: 50px;
+}
```
Comment (DeliceLydia) [inline src/App.css]: - [ ] As was suggested by the previous reviewer please follow the given design on [zeplin]() which means colors, layouts, etc
Screen Shot 2022-11-12 at 18 28 16

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Solobarine, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -3,17 +3,114 @@ body {
...
+  margin: 0;
+  padding: 0;
+  margin-bottom: 15px;
+}
+
+.bookDetails > p {
+  margin: 0;
+  padding: 0;
+  margin-top: 10px;
+}
```
Comment (codepantha) [inline src/App.css]: - [ ] You have done well by making some changes but just as the previous reviewer suggested, the project still needs some changes in order to look like the Zeplin design guideline. Please take note of font colors, buttons, margins, shadows etc. Please take a look at the attached images for more information 👇 
## Zeplin Design
image

## Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Solobarine 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Solobarine 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Solobarine** [comment]
Thank you very much.

**Pull request merged**